src/test/java/jimmy/mcgymmy/model/ModelManagerTest.java

```diff
@@ -126,6 +128,54 @@ public void canUndo_modelGotChangedButUndoAlready_returnsFalse() {
         assertFalse(modelManager.canUndo());
     }
 
+    @Test
+    public void clear_empty_mcGymmyHasCorrectContent() {
+        McGymmy expected = new McGymmyBuilder().build();
+        modelManager.clearFilteredFood();
+        assertEquals(expected, modelManager.getMcGymmy());
+    }
+
+    @Test
+    public void clear_nonEmpty_mcGymmyHasCorrectContent() {
+        McGymmy expected = new McGymmyBuilder().build();
+        modelManager.addFood(getChickenRice());
+        modelManager.addFood(getNasiLemak());
+        modelManager.clearFilteredFood();
+        assertEquals(expected, modelManager.getMcGymmy());
+    }
+
+    @Test
+    public void fnd_empty_mcGymmyHasCorrectContent() {
```

Comment:
I think there's a typo here and in the test method names below; should be `find_empty....` instead of `fnd_...`. Just a minor nit.

Reply:
ok!